app/frontend/app/templates/group/manage.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <div class="col-sm-12">
+    {{#if currentMember.isAdmin}}
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            Group Settings
+          </div>
+          <div class="panel-body">
+            {{#if model.closed}}
+              Your group is closed! You'll need to contact a staff member to re-open it.
+            {{else}}
+              <button class="btn btn-danger" {{action "closeGroup"}}>
+                Close Group
+              </button>
```

Comment:
It will be restricted, but the question is whether we ought to maintain this page.  I'm thinking we should just swap it for a "delete" button on the group page and then `if (prompt('Type the group slug to confirm') === this.get('model.id')) this.get('model').destroy();` or something — we can totally get away with using `prompt` since it's administrative stuff.